src/ui/otb/clockSet.ts

```diff
@@ -0,0 +1,74 @@
+import settings from '../../settings'
+import HandicapIncClock from '../shared/clock/clocks/HandicapIncClock'
+import DelayClock from '../shared/clock/clocks/DelayClock'
+import BronsteinClock from '../shared/clock/clocks/BronsteinClock'
+import HourglassClock from '../shared/clock/clocks/HourglassClock'
+import StageClock from '../shared/clock/clocks/StageClock'
+
+
+const MILLIS = 1000
```

Comment:
Should be imported from `../shared/clock`

Reply:
Agreed. Done.